monitor.html

```diff
@@ -0,0 +1,366 @@
+								var nacks_per_second = e.data.remoteStats[UUID].nacks_per_second;
+								updateData("nackrate", nacks_per_second, UUID);
+							} else if (document.getElementById(UUID)){
+								updateData("nackrate", 0, UUID);
+							}
+							
+							if (e.data.remoteStats[UUID].retransmitted_kbps){
+								var retransmitted_kbps = e.data.remoteStats[UUID].retransmitted_kbps;
+								updateData("retransmit", retransmitted_kbps, UUID);
+							} else if (document.getElementById(UUID)){
+								updateData("retransmit", 0, UUID);
+							}
+							
+							
+							//updateData("buffer", buffer);
+							
+							//if (packetloss != undefined) {
+							//	packetloss = packetloss.toFixed(2);
+							//	updateData("packetloss", packetloss);
+							//}
+						}
+					}
+				});
+			}
+
+			function printValues(obj) {
+				var out = "";
+				for (var key in obj) {
+					if (typeof obj[key] === "object") {
+						out += "<br />";
+						out += printValues(obj[key]);
+					} else {
+						if (key.startsWith("_")) {
+						} else {
+							out += "<b>" + key + "</b>: " + obj[key] + "<br />";
+						}
+					}
+				}
+				return out;
+			}
+
+			function logData(type, data, UUID) {
+				try{
+					var log = document.getElementById(UUID).querySelectorAll('[data-log]')[0].getElementsByTagName("ul")[0];
+					var entry = document.createElement('li');
+					entry.style.color ="white";
+					entry.textContent =	"[" + new Date().toLocaleTimeString() + "] " + type + " : " + data;
+					log.prepend(entry);
+				} catch(e){}
+			}
+		</script>
+	</head>
+	<body onload="loadIframe();">
+		<div id="container">
+			<h1>
+				OBS.Ninja Remote Monitor
+			</h1>
+		</div>
+		
+		<div id="graphs" style="display:none">
+		
+			<div data-log="true" onclick="copyFunction(this.innerText)" style="max-height:400px;display:inline-block;">
+				<ul></ul>
+			</div>
+			
+			<div class="graph">
+				<h2>Bitrate (kbps)</h2>
+				<span>0</span>
+				<canvas data-bitrate-graph="true"></canvas>
+			</div>
+
+			<div class="graph">
+				<h2>Nacks (per second)</h2>
+				<span>0</span>
+				<canvas  data-nackrate-graph="true"></canvas>
+			</div>
+
+			<div class="graph">
+				<h2>Retransmitted (kbps)</h2>
+				<span>0</span>
+				<canvas data-retransmit-graph="true"></canvas>
+			</div>
+		</div>
+		
+		
+
+
+		<script>
+			var bitrate = {
+				element: "bitrate-graph",
+				data: 0,
+				max: 6000,
+				target: 3000,
+			};
+			var frames;
+			var nackrate = {
+				element: "nackrate-graph",
+				data: 0,
+				max: 15,
+				target: 15,
+			};
+			var retransmit = {
+				element: "retransmit-graph",
+				data: 0,
+				max: 100,
+				target: 100,
+			};
+
+			function updateData(type, data, UUID) {
+				if (type == "bitrate") {
+					bitrate.data = data;
+					plotData("bitrate", bitrate, UUID);
+				}
+
+				if (type == "nackrate") {
+					nackrate.data = data;
+					plotData("nackrate", nackrate, UUID);
+				}
+
+				if (type == "retransmit") {

```

Comment:
It would probably be safer to use [eqeqeq](https://eslint.org/docs/rules/eqeqeq)
```suggestion
			function updateData(type, data, UUID) {
				if (type === "bitrate") {
					bitrate.data = data;
					plotData("bitrate", bitrate, UUID);
				}

				if (type === "nackrate") {
					nackrate.data = data;
					plotData("nackrate", nackrate, UUID);
				}

				if (type === "retransmit") {

```

Reply:
Thank you